launcher/launcher/main.go

```diff
@@ -130,25 +144,35 @@ func getExitCode(isHardened bool, restartPolicy spec.RestartPolicy, err error) i
 
 func startLauncher() error {
 	logger.Println("Launch Spec: ", launchSpec)
-	client, err := containerd.New(defaults.DefaultAddress)
+	containerdClient, err := containerd.New(defaults.DefaultAddress)
 	if err != nil {
 		return &launcher.RetryableError{Err: err}
 	}
-	defer client.Close()
+	defer containerdClient.Close()
 
 	tpm, err := tpm2.OpenTPM("/dev/tpmrm0")
 	if err != nil {
 		return &launcher.RetryableError{Err: err}
 	}
 	defer tpm.Close()
 
+	// check EK cert
+	gceAk, err := client.GceAttestationKeyECC(tpm)
+	if err != nil {
+		return &launcher.RetryableError{Err: err}
+	}
+	if len(gceAk.CertDERBytes()) == 0 {
```

Comment:
you can just do `if gceAk.Cert() == nil`

Reply:
Done